src/vstart.sh

```diff
@@ -1302,6 +1302,14 @@ EOF
                     debug echo dashboard module not working correctly!
                 fi
             fi
+
+            ceph_adm osd pool create rbd
+            ceph_adm osd pool application enable rbd rbd
+
+            if [ -n "${NVMEOF_GW}" ]; then
+                echo "Adding nvmeof-gateway ${NVMEOF_GW} to dashboard"
+                ceph_adm dashboard nvmeof-gateway-add -i <(echo "${NVMEOF_GW}") "${HOSTNAME}"
```

Comment:
$HOSTNAME will be the name of this host/container... The `${NVMEOF_GW/:/_}` will replace `:` with `_` for the ports.
```suggestion
                ceph_adm dashboard nvmeof-gateway-add -i <(echo "${NVMEOF_GW}") "${NVMEOF_GW/:/_}"
```